### PR TITLE
feat: new main "Login" page design

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

A new design for the main "Login" page that should add more links to content outside the app but also make to the inner routes navigation links more obvious and other designer thingy.

# PRs

- [X] gix-cmp https://github.com/dfinity/gix-components/pull/140

# Changes

- new components and assets for login screen new design
- no functional changes
- few components renamed and refactored

# Screenshots

Capture d’écran 2022-12-21 à 11 30 45
Capture d’écran 2022-12-21 à 11 30 51
Capture d’écran 2022-12-21 à 11 31 02
Capture d’écran 2022-12-21 à 11 31 09
Capture d’écran 2022-12-21 à 11 31 37
Capture d’écran 2022-12-21 à 11 31 48
Capture d’écran 2022-12-21 à 11 32 45
Capture d’écran 2022-12-21 à 11 33 41
Capture d’écran 2022-12-21 à 11 33 45

### CODE REVIEW

File: frontend/src/lib/i18n/en.json [hunk 1]
```diff
@@ -125,15 +125,20 @@
...
-    "ic": "INTERNET COMPUTER",
-    "nns": "NETWORK NERVOUS SYSTEM",
     "login": "Sign in with Internet Identity",
     "title": "Open and autonomous governance running",
     "on_chain": "100% on-chain",
-    "wallet": "ICP WALLET",
-    "stake": "STAKE ICP NEURONS",
-    "earn": "VOTE & EARN REWARDS",
-    "ic_logo": "Internet Computer logo"
+    "wallet": "ICP Wallet",
```
Comment (mstrasinskis) [inline frontend/src/lib/i18n/en.json:131]: Should the texts be sync with the main menu update? And not so emphasise purely `ICP` token?

File: frontend/src/lib/pages/Login.svelte [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  const unsubscribe: Unsubscriber = authStore.subscribe(
+    async ({ identity }: AuthStore) => {
+      signedIn = isSignedIn(identity);
+
+      if (!signedIn) {
+        return;
+      }
+
+      // TODO: to be removed
+      // Backwards compatibility until the dashboard has migrated to the new query parameters URL
```
Comment (mstrasinskis) [inline frontend/src/lib/pages/Login.svelte:28]: Is there a dashboard jira task for the migration?

File: frontend/src/routes/(login)/+layout.svelte [hunk 3]
```diff
@@ -2,45 +2,104 @@
...
     display: flex;
     flex-direction: column;
-    height: 100%;
-    background: transparent;
-    overflow-y: auto;
+    justify-content: flex-end;
+    align-items: center;
+
+    min-height: calc(
+      100vh - var(--login-header-height) - (2 * var(--padding-4x)) -
```
Comment (mstrasinskis) [inline frontend/src/routes/(login)/+layout.svelte:83]: `(2 * var(--padding-4x))` == `var(--padding-8x)` ?

**Review by mstrasinskis — APPROVED** [review]
LGTM, just a couple of questions.

File: frontend/src/lib/pages/Login.svelte (hunk 2 above)
Comment (peterpeterparker) [reply frontend/src/lib/pages/Login.svelte:28]: There is a jira task on the dashboard side. Don't remember the number but I subscribed to it.

File: frontend/src/routes/(login)/+layout.svelte (hunk 3 above)
Comment (peterpeterparker) [reply frontend/src/routes/(login)/+layout.svelte:83]: You know math are complicated sometimes 😉. Thanks!

File: frontend/src/lib/i18n/en.json (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/i18n/en.json:131]: I'll ask. Let's merge and provide a textual PR  change afterwards if needed.

**Pull request merged**